backend/danswer/connectors/alation/connector.py

```diff
@@ -0,0 +1,228 @@
+import requests
```

Comment:
nit: if you set the precommit hooks or run reorder-python-imports, I believe this one gets put under third party imports right before bs4

https://github.com/danswer-ai/danswer/blob/main/CONTRIBUTING.md#formatting-and-linting